test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/ActivityExtensionsTest.cs

```diff
@@ -207,6 +209,8 @@ public void ToOtlpSpanTest()
             Assert.Equal(OtlpTrace.Span.Types.SpanKind.Client, otlpSpan.Kind);
             Assert.Equal(traceId, otlpSpan.TraceId);
             Assert.Equal(parentId, otlpSpan.ParentSpanId);
+            Assert.Equal(OtlpTrace.Status.Types.StatusCode.NotFound, otlpSpan.Status.Code);
+            Assert.Equal(Status.NotFound.Description ?? string.Empty, otlpSpan.Status.Message);
```

Comment:
Just curious, isn't the description always null for the static Status members?

Reply:
We could set description to status by calling `WithDescription` method.
`Status.Ok.WithDescription("Success")`

Tests related to these are covered in #802